rank-info/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "slippi-rank-info"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { workspace = true }
+tracing = { workspace = true }
+dolphin-integrations = { path = "../dolphin" }
+serde_json = { workspace = true }
+serde_repr = { workspace = true }
+thiserror = "1.0.44"
```

Comment:
`jukebox` uses `thiserror` as well, so we should move this to the workspace `Cargo.toml` and have both projects reference the workspace version.